boards/arm/qemu_cortex_r5/qemu_cortex_r5.dts

```diff
@@ -13,10 +13,11 @@
 	compatible = "xlnx,zynqmp-qemu";
 
 	chosen {
-		zephyr,sram = &sram0;
-		zephyr,flash = &flash0;
-		zephyr,console = &uart0;
+		zephyr,sram       = &sram0;
+		zephyr,flash      = &flash0;
+		zephyr,console    = &uart0;
```

Comment:
Not sure if this is the best thing to do; sure it looks nice, but such attempts can easily get out of hand and does not necessarily help with code readability.

Reply:
reverted the indentation changes.